src/plugins/wb-jsonmanager/jsonmanager.js

```diff
@@ -273,11 +378,57 @@ var componentName = "wb-jsonmanager",
 						}
 					} else {
 
-						// Do an empty fetch to ensure jsonPointer is loaded and correctly initialized
-						$elm.trigger( {
-							type: "json-fetch.wb"
-						} );
-						wb.ready( $elm, componentName );
+						if ( elmData.srcExtractor && Array.isArray( elmData.srcExtractor ) ) {
+
+							extractData( elmData );
+
+							var settings = wb.getData( $elm, componentName );
+
+							dsName = "[" + settings.name + "]";
+
+							datasetCache = jsonSource[ elmData.name ];
+							var dsJSON = jsonSource[ elmData.name ];
+							var delayedLst = dsDelayed[ dsName ] || [];
+							i_len = delayedLst.length;
+							for ( i = 0; i !== i_len; i += 1 ) {
+								i_cache = delayedLst[ i ];
+								var pntrSelector = i_cache.selector;
+								var resultSet = {};
+								if ( pntrSelector.length ) {
+									try {
+										resultSet = jsonpointer.get( dsJSON, pntrSelector );
+									} catch  ( e ) {
+										throw dsName + " - JSON selector not found: " + pntrSelector;
+									}
+								} else {
+									resultSet = dsJSON;
+								}
+								$( "#" + i_cache.callerId ).trigger( {
+									type: "json-fetched.wb",
+									fetch: {
+										response: resultSet,
+										status: "200",
+										refId: i_cache.refId,
+										xhr: null
+									}
+								}, this );
+							}
+
+							$elm.trigger( {
+								type: "json-fetch.wb"
+							} );
```

Comment:
Défini le jsonSource directement à la place de le faire via une globale.

Par example:
```suggestion
							$elm.trigger( {
								type: "json-fetch.wb",
								fetch: {
									response: jsonSource
								}
							} );
```

Reply:
Applied. Thanks